src/pages/Products/Products.js

```diff
@@ -41,6 +50,7 @@ export function Products() {
               </div>
             ))}
         </div>
+        <Pagination onPageChange={onPageChange} />
```

Comment:
you can simply pass referense to setCurrentPage function here instead of creating onPageChange handler that just passes pageNumber to setter function

Reply:
also, it would be nice to scroll the page to the top of the list when the cuurent page is changing